tests/requirements/dev.txt

```diff
@@ -1,3 +1,4 @@
-django-celery==3.0.23
-djtables==0.1.2
-mock==1.0.1
+django-celery>=3.0.23
```

Comment:
I think there were some changes to celery APIs in 3.1 so we might want to make this `django-celery>=3.0.23,<3.1` ?  And similarly wherever we require celery itself.